### DISCUSSION TITLE
T024: Plot sequence logo from Python

### DISCUSSION THREAD

**Discussion by dominiquesydow** [volkamerlab/teachopencadd#268] [Ideas]
I just came across this while searching for something else - it seems like there is an option to plot sequence logos from Python :tada:

https://www.biotite-python.org/examples/gallery/sequence/anderson_logo.html#sphx-glr-examples-gallery-sequence-anderson-logo-py

Pinging @t-kimber because we were wondering about this when working on the current version of T024.

**Comment by dominiquesydow** [comment]
Oh my, this whole `biotite` example collection is a gold mine:
https://www.biotite-python.org/examples/gallery/index.html

@AndreaVolkamer for the future, worth thinking about using some of these visualizations in TeachOpenCADD talktorials (could be smaller student projects).
- Ramachandran plots: https://www.biotite-python.org/examples/gallery/structure/ramachandran.html#sphx-glr-examples-gallery-structure-ramachandran-py
- Pairwise sequence alignment with nice visualization: https://www.biotite-python.org/examples/gallery/sequence/avidin_alignment.html#sphx-glr-examples-gallery-sequence-avidin-alignment-py
- More sequence alignment: https://www.biotite-python.org/examples/gallery/sequence/pi3k_alignment.html#sphx-glr-examples-gallery-sequence-pi3k-alignment-py
- Superposition (thinking of OpenCADD's superposition module): https://www.biotite-python.org/examples/gallery/structure/ku_superimposition.html#sphx-glr-examples-gallery-structure-ku-superimposition-py

Pinging @padix-key to say thanks for this fantastic package and tutorials!!

**Comment by padix-key** [comment]
> Pinging @padix-key to say thanks for this fantastic package and tutorials!!

Thank you very much!
